### PR TITLE
Add Hero plugin translations

### PR DESCRIPTION
**Pull request by yesszus**

### CODE REVIEW

File: e107_plugins/hero/languages/English/English_admin.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+|        e107 website content management system Hungarian Language File
+|        Released under the terms and conditions of the
+|        GNU General Public License (http://gnu.org).
+|        Last Modified: 2020/12/18 12:42:44
+|
+|        $Author: Yesszus $
++----------------------------------------------------------------------+
+*/
+
+define("LAN_HERO_SUCC", "Default slides imported.");
```
Comment (CaMer0n) [inline e107_plugins/hero/languages/English/English_admin.php:13]: @yesszus Thank you for this! It's great, it just needs to use the correct naming standard. 

```
LAN_HERO_ADMIN_001
LAN_HERO_ADMIN_002
...

```

File: e107_plugins/hero/languages/English/English_global.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+|        e107 website content management system Hungarian Language File
+|        Released under the terms and conditions of the
+|        GNU General Public License (http://gnu.org).
+|        Last Modified: 2020/12/18 12:42:44
+|
+|        $Author: Yesszus $
++----------------------------------------------------------------------+
+*/
+
+define("LAN_PLUGIN_HERO_DIZ", "Image and text slider, animated bullet points for the hero area of your home page.");
```
Comment (CaMer0n) [inline e107_plugins/hero/languages/English/English_global.php:13]: @yesszus This file is correct. Only LANs used globally. (such as "all of the admin area" or "all of frontend") use the non-numeric naming standard.

**Comment by CaMer0n** [comment]
@yesszus Thank you again. I'm not sure if you saw my earlier comments. 

To summarize - here's the correct naming standard for new LAN files: 

### English_front.php:

    LAN_HERO_001 etc. 

### English_admin.php

    LAN_HERO_ADMIN_001

### English_menu.php

    LAN_HERO_MENU_001

### English_global.php

    LAN_PLUGIN_HERO_NAME (used by plugin.xml and elsewhere) 
    LAN_PLUGIN_HERO_DESCRIPTION (used by plugin.xml and elsewhere) 

Additional 'named' LANs may be added here (if they they need to be available to every page of the site)
using the same format:
     
    LAN_PLUGIN_HERO_XXXX
   
@Moc @realmontazeri For the docs. 


> @yesszus Thank you for this! It's great, it just needs to use the correct naming standard.
> 
> ```
> LAN_HERO_ADMIN_001
> LAN_HERO_ADMIN_002
> ...
> ```

File: e107_plugins/hero/templates/hero_template.php [hunk 3]
```diff
@@ -53,7 +53,7 @@
  											</div>
 					                      <div class="hero-buttons text-right">
 					     
-					                          <a href="{HERO_BUTTON1_URL}" class="btn btn-{HERO_BUTTON1_CLASS} btn-raised animated fadeInRight animation-delay-28">
+					                          <a target="_blank" href="{HERO_BUTTON1_URL}" class="btn btn-{HERO_BUTTON1_CLASS} btn-raised animated fadeInRight animation-delay-28">
```
Comment (CaMer0n) [inline e107_plugins/hero/templates/hero_template.php:56]: @yesszus Why this change?

Comment (yesszus) [reply e107_plugins/hero/templates/hero_template.php:56]: Content that points to an external link do not leave the website.

Comment (CaMer0n) [reply e107_plugins/hero/templates/hero_template.php:56]: @yesszus Thanks. But we do not know for sure that it is an external link.

Comment (yesszus) [reply e107_plugins/hero/templates/hero_template.php:56]: Ok.
